projects/plugins/boost/app/features/optimizations/minify/Minify.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Features\Optimizations\Minify;
+
+use Automattic\Jetpack_Boost\Contracts\Feature;
+use Automattic\Jetpack_Boost\Features\Optimizations\Minify\Config;
+
+class Minify implements Feature {
+	// @todo - handle PHP constants.
+
+	public function setup() {
+		require 'functions-helpers.php';
+
+		// TODO: Make concat URL dir configurable
+		if ( isset( $_SERVER['REQUEST_URI'] ) && '/_static/' === substr( $_SERVER['REQUEST_URI'], 0, 9 ) ) {
+			require_once __DIR__ . '/service.php';
+			exit;
+		}
+
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$disable = isset( $_GET['jb-disable-minify'] ) || jetpack_boost_page_optimize_bail();
```

Comment:
There is already a GET parameter; `jb-disable-modules` which lets you pass in a list of module slugs to disable (or `all` for all). Do we need a new GET param?

Reply:
I wasn't aware of that GET param - thanks.

Updated with [f477e23](https://github.com/Automattic/jetpack/pull/28686/commits/f477e237a3c15971782f0ef61930ca091bf3ca13).